packages/mock-app/src/constants/app-config.example.json

```diff
@@ -413,8 +422,17 @@
                     "dlrAPIKey": "5555555555555"
                   },
                   "storage": {
-                    "storageAPIUrl": "https://storage.example.com",
-                    "bucket": "bucket-stored-example"
+                    "url": "http://localhost:3333/v1/credentials",
```

Comment:
shall we update the readme to remind the implementers that they need to run the local storage server first?